src/Magento/FunctionalTestingFramework/Allure/Adapter/MagentoAllureAdapter.php

```diff
@@ -354,4 +359,21 @@ private function retrieveStepKey($stepLine)
 
         return $stepKey;
     }
+
+    /**
+     * Removes attachments from step depending on MFTF configuration
+     * @param Step    $step
+     * @param Failure $testFailed
+     * @return void
+     */
+    private function removeAttachments($step, $testFailed)
+    {
+        //Remove Attachments if verbose flag is not true AND test did not fail
+        if (getenv('VERBOSE_ARTIFACTS') !== true && $testFailed === null) {
```

Comment:
Please check this condition again after address earlier comment.

Reply:
Fixed